src/style.css

```diff
@@ -1 +1,45 @@
 /* add styles here */
+html{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the `*` selector for styling. It impacts performance as it affects all elements. Try to set styles only for elements that require them.